DMTN-107.tex

```diff
@@ -178,28 +180,36 @@ \subsection{Do Nothing}\label{ssec:potsol_donothing}
+Although not strictly speaking a de-scope, this option is likely be perceived as a de-scope by the community and will fall below their expectations. 
+
+As an additional note, LSST Education and Public Outreach (EPO) are not expecting to serve alerts to the community prior to Data Release 1 and the beginning of LSST Alert Stream production \citedsp{LEP-031}, and so this option would not have any scope implications for EPO. 
 
 \textcolor{orange}{ Risks -- Potential risks for the DMS.}
-There is a risk to DM in missing the opportunity to learn lessons from running alert production in LOY1.
-This puts a higher importance on running alert production during commissioning, and/or imposes a higher risk on alert production when it finally does start in year 2.
-There is another risk to the DMS: it is likely that the community will do DIA and/or alert production with user-generated pipelines based on the processed standard-visit images released within 24 hours\reqparam{L1PublicT}\lsrreq{0104}.
+There is a risk to DM in missing the opportunity to learn lessons from running alert production in LOY1
+% LPG: I don't think the decision to not run AP in LOY1 will change anything during commissioning If anything, 
+% there will be less of a reason to produce templates during commissioning as they won't be used in LOY1. If commissioning gets squished, this will be an easier task to drop 
+% This puts a higher importance on running alert production during commissioning  
+and imposes a higher risk on alert production when it finally does start in year 2.
+There is another risk to the DMS: it is likely that the community will do ad-hoc template generation, DIA and/or alert production with user-generated pipelines based on the processed standard-visit images released within 24 hours\reqparam{L1PublicT}\lsrreq{0104} in order to get early science and publications.
 This might cause additional load on the user computational processing resources available through the Science Platform, especially if multiple user groups attempt to compete in this regard (which seems likely).
 
-\textcolor{red}{ Requirements -- Violates high-level SRD requirements.}
-As described in Section \ref{ssec:req_ap}, the SRD requires that alerts be generated continuously every observing night \citedsp{LPM-17}.
-While it is unavoidable that there be some areas without template images during LOY1, and thus also without alerts, the option to not generate {\it any} alerts in LOY1 is clearly in violation of the requirements (and would certainly violate the science community's expectations).
+% LPG Suggested rephrasing to soften the implication of a strict violation
+\textcolor{orange}{ Requirements -- Not consistent with high-level SRD requirements.}
+As described in Section \ref{ssec:req_ap}, of the SRD,  {\it rough guidance}, rather than a strict requirement,  states that alerts be generated continuously every observing night \citedsp{LPM-17}. 
```

Comment:
The term 'rough guidance' is a direct quote from the SRD - I will rephrase so that this is clear

Reply:
Update to: "As described in Section \ref{ssec:req_ap}, the SRD states that it provides {\it rough guidance}, rather than a strict requirement, that alerts be generated continuously every observing night \citedsp{LPM-17}."

Reply:
Resolved in the text with "As described in Section \ref{ssec:req_ap}, the SRD states that it provides {\it rough guidance}, rather than a strict requirement, that alerts be generated continuously every observing night \citedsp{LPM-17}."